src/routes/state-management/_components/ServerStateExample.tsx

```diff
@@ -59,6 +60,7 @@ function ServerStateExample() {
             width={'1/5'}
             onChange={key => {
               navigate({
+                // @ts-ignore
```

Comment:
why did you add these comments?

Reply:
Because I red the tanstack documentation which provides examples that looks exactly the same. But typescript moans about it that it doesn't exists and I didn't find out why. And at some point I thought I don't care. I also don't like it - and you shouldn't do it - but if you have an idea feel free to change it.

Reply:
https://tanstack.com/router/v1/docs/framework/react/guide/search-params#usenavigate-navigate-search-

Reply:
Btw I just hope it's goes away in another version^^